cmd/ci-operator-prowgen/main.go

```diff
@@ -451,7 +458,9 @@ func generateJobs(
 			if c := configSpec.ReleaseTagConfiguration; c != nil {
 				release = c.Name
 			}
-			if conf := element.OpenshiftInstallerRandomClusterTestConfiguration; conf != nil {
+			if element.OpenshiftInstallerRandomClusterTestConfiguration != nil {
+				podSpec = generatePodSpecRandom(info, &element)
+			} else if element.OpenshiftInstallerSrcRandomClusterTestConfiguration != nil {
```

Comment:
Both branches are the same, is this intentional? Looks correct, but then an OR instead of a separate `if` may be in order.

Reply:
It was supposed to keep the line short(er), using those names is a bit ridiculous. I'll change to an `||`.